addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -1208,6 +1210,17 @@ registerModel({
         channel: one('Thread', {
             inverse: 'rtc',
         }),
+        /**
+         * Call all sessions for which no peerConnection is established at
+         * a regular interval to try to recover any connection that failed
+         * to start.
+         *
+         * This is distinct from this._recoverConnection which tries to restores
+         * connection that were established but failed or timed out.
+         */
+        connectInterval: attr({
+            compute: '_computeConnectInterval',
+        }),
```

Comment:
I tested and it still works, but this comment is partially incomplete.

The timeout also does the server ping every 30s no matter what, which serves to ensure all new sessions are eventually known, even if they previously failed to fetch due to race conditions and other issues.

Reply:
It's currently only documenting the "_callSessions" part but not the "_pingServer" part. I think both should be documented the same.

Reply:
The ping is also useful to eject "timed out" members, and that's very not obvious from the code or the name itself.

Reply:
Just documented `ping`, and also renamed the interval to `pingInterval`, as the "ping" aspect is the most important one.
I really don't like the wording "call [peer/session]", so instead I use "connect to [RtcSession]".
This reduces the amount of concepts and makes it much easier to understand.

In follow-up PRs, most things will be in `RtcSession` and it's all actions on the `RtcSession`.
"Peer" will be replaced to "RtcSession" altogether. Just `RtcPeerConnection` will be technical detail of `RtcSession`, but we'll avoid using "Peer" in documentation and field naming.